### PR TITLE
Jb/hotfix 510, 524 and sleeping

### PR DESCRIPTION
**Pull request by justbill2020**
####  Description

this fixes the issue with #510, #524 and a issue from user on discord regarding smart-mirror not sleeping after specified amount of minutes.

####  Fixes Related issues
- #510 
- #524 

####  Checklist
- [x] I have read and understand the CONTRIBUTIONS.md file
- [x] I have searched for and linked related issues
- [x] I have added config.schema.json file if config option are required.
- [x] I am NOT targeting master branch

### CODE REVIEW

File: plugins/autosleep/controller.js [hunk 1]
```diff
@@ -8,19 +8,10 @@ function AutoSleep($scope, $http, $q, SpeechService,AutoSleepService) {
...
-	    console.debug('controller Wake');
-	    AutoSleepService.stopAutoSleepTimer();
-    }); 
-    $scope.$on('autoSleep.sleep', function(e, value) {
-        AutoSleepService.sleep();
-        $scope.$parent.focus = AutoSleepService.scope; 
-	    console.debug('controller sleep');
+    $scope.$on('refreshScopeFocus', function (e, value) {
+        $scope.$parent.focus = value
     });
```
Comment (evancohen) [inline plugins/autosleep/controller.js]: You should be using Focus to change the focus (occurs in multiple places in this PR), `$scope.$parent.focus` does weird things. Working example:
https://github.com/evancohen/smart-mirror/blob/dev/plugins/search/controller.js#L29

Comment (justbill2020) [inline plugins/autosleep/controller.js]: Can I do that from a service? Or only a controller?

Comment (justbill2020) [inline plugins/autosleep/controller.js]: I dig the new focus service

File: plugins/_general/config.schema.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               {
```
Comment (justbill2020) [inline plugins/_general/config.schema.json]: no not even an issue... its white space... blank line above bracket on new  blank line after on HEAD...

**Pull request merged**